iocage.8

```diff
@@ -1015,6 +1046,30 @@ The Source listed with each property shows whether it is a local
 .Nm
 property or where more information can be located.
 .Bl -tag -width "pkglist=none"
+.It Pf assign_localhost= Op 0 | 1
+Boolean option to add interface
+.Ql lo0
+assigned to
+.Ql 127.0.0.1 .
```

Comment:
It starts at 127.0.0.2 and works up tot he next available address. 127.0.0.1 is not assigned.